screens/players/components/player-recent-matches-tab.tsx

```diff
@@ -88,7 +88,7 @@ const PlayerRecentMatchesTab = ({
     const matchTypeMap: { [key: number]: FilterInformation } = {};
     playerMatchesData?.forEach(({ mapname, matchtype_id }) => {
       mapNameMap[mapname] = {
-        label: maps[mapname]?.name || mapname,
+        label: isOfficialMap(mapname) ? maps[mapname]?.name : mapname,
```

Comment:
Btw I am not sure if we need the OfficialMapKeys, because the previous code did the same thing `maps[mapname]?.name` would return undefined if the key was not in `maps` object. So basically the same thing would happen. 

But I guess OfficialMapKeys could be useful in future / for example if it's not official map name. We could render a placeholder image for the map image.

Reply:
Ah I see, you added it as typescript types. OK that make sense 👍